docs/dictionary.rst

```diff
@@ -25,6 +25,7 @@ Dockstore Dictionary
 	* :ref:`dict container`
 	* :ref:`dict Cromwell`
 	* :ref:`dict CWL`
+	* :ref:`dict cwltool`
```

Comment:
I wish we could .gitignore this file, and maybe we can. We'd have to check how the builds work and whatnot.

No action required, just noting. But I started to review this file before realizing I'd already reviewed the files it's generated from. It's a generated file, so it shouldn't show up in reviews.

Reply:
Think I found a solution: https://github.com/dockstore/dockstore-documentation/pull/231